src/html5_parser/parser/adoption_agency.rs

```diff
@@ -12,13 +12,15 @@ pub enum AdoptionResult {
     Completed,
 }
 
-impl<'a> Html5Parser<'a> {
-    /// When we talk about nodes, there are 3 contexts to consider:
-    ///
-    /// - The actual node data. This is called "node" in the code.
-    /// - The node id. This is called "node_id" in the code.
-    /// - The node index. This is called "node_idx" in the code. This is the index of the node in  
-    /// either the open_elements or active_formatting_elements stack.
+impl<'stream> Html5Parser<'stream> {
+    /**
+     * When we talk about nodes, there are 3 contexts to consider:
+     *
+     * - The actual node data. This is called "node" in the code.
+     * - The node id. This is called "node_id" in the code.
+     * - The node index. This is called "node_idx" in the code. This is the index of the node in
+     *   either the open_elements or active_formatting_elements stack.
+     */
     pub fn run_adoption_agency(&mut self, token: &Token) -> AdoptionResult {
```

Comment:
We should probably do a second pass soon and reintroduce some of the good changes, like this one, that are being reverted here (not necessarily in this PR).

Reply:
Agreed. I think that some of the reverts are due to a lot of manual fixups during rebasing of this branch on main, as there were a lot of changes done in both branches. This should (hopefully) not happen very often as codebase matures